deployment/terraform/container_service.tf

```diff
@@ -0,0 +1,201 @@
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_lb_listener" "oar" {
+  load_balancer_arn = "${aws_lb.oar.id}"
+  port              = "443"
+  protocol          = "HTTPS"
+  certificate_arn   = "${module.cert.arn}"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.oar.id}"
+    type             = "forward"
+  }
+}
+
+#
+# ECS Resources
+#
+
+resource "aws_ecs_cluster" "oar" {
+  name = "ecsCluster${title(var.project)}${title(var.environment)}"
+}
+
+data "template_file" "oar_ecs_task" {
+  template = "${file("task-definitions/oar.json")}"
+
+  vars = {
+    cpu    = "${var.fargate_cpu}"
+    image  = "${local.app_image}"
+    memory = "${var.fargate_memory}"
+
+    postgres_host     = "${module.database.hostname}"
+    postgres_port     = "${module.database.port}"
+    postgres_user     = "${var.rds_database_username}"
+    postgres_password = "${var.rds_database_password}"
+    postgres_db       = "${var.rds_database_name}"
+
+    django_secret_key = "${var.django_secret_key}"
+
+    app_port = "${var.app_port}"
+
+    aws_region = "${var.aws_region}"
+
+    project     = "${var.project}"
+    environment = "${var.environment}"
+  }
+}
+
+resource "aws_ecs_task_definition" "oar" {
+  family                   = "oar"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "${var.fargate_cpu}"
+  memory                   = "${var.fargate_memory}"
+
+  execution_role_arn = "${aws_iam_role.ecs_task_execution_role.arn}"
+
+  container_definitions = "${data.template_file.oar_ecs_task.rendered}"
+}
+
+data "template_file" "oar_management_ecs_task" {
+  template = "${file("task-definitions/oar_management.json")}"
+
+  vars = {
+    cpu    = "${var.fargate_cpu}"
+    image  = "${local.app_image}"
+    memory = "${var.fargate_memory}"
+
+    postgres_host     = "${module.database.hostname}"
+    postgres_port     = "${module.database.port}"
+    postgres_user     = "${var.rds_database_username}"
+    postgres_password = "${var.rds_database_password}"
+    postgres_db       = "${var.rds_database_name}"
+
+    django_secret_key = "${var.django_secret_key}"
+
+    app_port = "${var.app_port}"
+
+    aws_region = "${var.aws_region}"
+
+    project     = "${var.project}"
+    environment = "${var.environment}"
+  }
+}
+
+resource "aws_ecs_task_definition" "oar_management" {
+  family                   = "mgmt${title(var.project)}${title(var.environment)}"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "${var.fargate_cpu}"
+  memory                   = "${var.fargate_memory}"
+
+  execution_role_arn = "${aws_iam_role.ecs_task_execution_role.arn}"
+
+  container_definitions = "${data.template_file.oar_management_ecs_task.rendered}"
+}
+
+resource "aws_ecs_service" "oar" {
+  name            = "ecsService${title(var.project)}${title(var.environment)}"
```

Comment:
This name does not surface what the service is. I think including that in the name is more important than `ecsService` and project name.